requirements-dev.txt

```diff
@@ -2,3 +2,4 @@ pyyaml==4.2b1
 jsonschema==2.6.0
 python-slugify==2.0.1
 exabyte_json_include==2.0.0
+-e .
```

Comment:
I suggest explaining in a comment:

"""
-e means "editable", meaning that pip is linking the source code into the venv directory, rather than copying it.  This way you can continue to develop it without reinstalling it every time
"""

or adding a docs link

Reply:
We're going to be doing this in every single python package.  Maybe it'd be enough to just have a confluence document that contains some notable python best practices?

Reply:
Sure